src/Illuminate/Support/Collection.php

```diff
@@ -1693,6 +1693,11 @@ public static function proxy($method)
         static::$proxies[] = $method;
     }
 
+    public function getProxies()
+    {
+        return static::$proxies;
```

Comment:
I had to add this to get access to the proxies. Would it being made public be preferable? I would assume not.